internal/agent/config.go

```diff
@@ -7,35 +7,60 @@ import (
 	"path/filepath"
 	"time"
 
-	"github.com/flightctl/flightctl/internal/util"
 	"k8s.io/klog/v2"
 	"sigs.k8s.io/yaml"
 )
 
 const (
 	// DefaultFetchSpecInterval is the default interval between two reads of the remote device spec
 	DefaultFetchSpecInterval = time.Second * 60
-
 	// DefaultStatusUpdateInterval is the default interval between two status updates
 	DefaultStatusUpdateInterval = time.Second * 60
+	// DefaultDataDir is he default directory where the device's data is stored
+	DefaultDataDir = "/etc/flightctl/"
```

Comment:
Would `DefaultConfigDir` be more appropriate, since it's `/etc`?
Read-write data would instead go into `/var/lib/flightctl`, as ostree leaves that alone when merging.

Reply:
makes sense to me this was the existing value will update